profile/nodejs.sh

```diff
@@ -24,10 +29,13 @@ detect_memory() {
 
 export PATH="$HOME/.heroku/node/bin:$HOME/bin:$HOME/node_modules/.bin:$PATH"
 export NODE_HOME="$HOME/.heroku/node"
-export NODE_ENV=${NODE_ENV:-production}
 
 calculate_concurrency
 
 export MEMORY_AVAILABLE=$MEMORY_AVAILABLE
 export WEB_MEMORY=$WEB_MEMORY
 export WEB_CONCURRENCY=$WEB_CONCURRENCY
+
+if [ "$LOG_CONCURRENCY" = "true" ]; then
```

Comment:
I have another buildpack which expects node to be installed via this webpack. On circlci deployment is failing with the following variable.

`remote: /tmp/build_f9481d5b2938229e585365cbf1fce652/.profile.d/nodejs.sh: line 39: LOG_CONCURRENCY: unbound variable`

Once I set LOG_CONCURRENCY explicitly in heroku settings the error dissapeared. So to me this should be added to the documentation or set via environment.sh if it does not exist.